wireapp/wire-android
# feat: Highlight selected searched message (WPB-5506)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-5506" title="WPB-5506" target="_blank"><img alt="Task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10818?size=medium" />WPB-5506</a>  [Android] add a blue transparent layer on the search result after navigating to it
  </td></table>
  <br />
 














----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

When a searched message is selected and user was navigated to Conversation screen, there was no indication of which message was selected and where it was placed.

### Causes (Optional)

Not implemented.

### Solutions

Add highlight of selected message in UI (in Conversation screen) with a blue transparent layer on a animation of 2 seconds.

### Testing

#### How to Test

- Open App
- Open Conversation
- Open Conversation Details
- Open Search Screen
- type a message you want to search
- from the results, select a message
- when navigated to the Conversation screen, you should see the highlighting animation of 2 seconds on the selected message.

### Attachments (Optional)


https://github.com/wireapp/wire-android-reloaded/assets/5890660/3b630cf2-7586-4360-93c3-5eb05e9a1786
